### ISSUE TITLE
Driver-only curves: parity starter

### ISSUE THREAD

**Issue by mpg** [Mbed-TLS/mbedtls#7344]
**Context:** see #6839.

**Depends on:** #7103

- [ ] Create a reference component associated with the two components introduced in #7103 (unless I'm mistaken, they can share the same reference component).
- [ ] Add parity testing in `analyze_outcomes.py` for the two pairs ("only p192" vs ref, "only x25519" vs ref) - for now add as many "ignore" entries as needed to make it pass.
- [ ] Do a local trial run of `ssl-opt.sh` (with parity) to get a feel of how it goes.
- [ ] Adjust follow-up issues (might result in: closing as there's nothing to do, editing the description to reflect work to be done, changing the size estimate to "s" is there's little work to be done, splitting into multiple issues if there's a large amount of work, etc.) according to findings.

Note: this is similar to #6855, #7142 and #7255 except in this case there are two driver components and they have been added in #7103 already.

**Comment by valeriosetti** [comment]
> unless I'm mistaken, they can share the same reference component

I agree ;)

**Comment by valeriosetti** [comment]
> Do a local trial run of ssl-opt.sh (with parity) to get a feel of how it goes

I tried a quick run of `ssl-opt.sh` and it seems to me that a specific PR will be required since tests are failing since the very beginning with: `SERVER START TIMEOUT`

**Comment by mpg** [comment]
Closing as there's been a change of strategy, making this no longer relevant (for the time being at least), see https://github.com/Mbed-TLS/mbedtls/issues/7241#issuecomment-1493931720
